isis/src/base/objs/Camera/Camera.cpp

```diff
@@ -1540,9 +1540,9 @@ namespace Isis {
       // order of points in vector is top, bottom, left, right
       QList< QPair< double, double > > surroundingPoints;
       surroundingPoints.append(qMakePair(samp, line - 0.5));
-      surroundingPoints.append(qMakePair(samp, line + 0.5 - DBL_MIN));
+			surroundingPoints.append(qMakePair(samp, std::nexttoward(line + 0.5, line)));
```

Comment:
Looks like a weird spacing issue is happening here. Check your tabs vs spaces in your editor.